resources/scripts/ttx_diff.py

```diff
@@ -13,12 +13,34 @@
     # rebuild the fontc copy, reuse the prior fontmake copy (if present), and compare
     # useful if you are making changes to fontc meant to narrow the diff
     python resources/scripts/ttx_diff.py --rebuild fontc ../OswaldFont/sources/Oswald.glyphs
+
+JSON:
+    If the `--json` flag is passed, this tool will output JSON.
+
+    If the script exits with a `0` status code, then `stdout` is a JSON dictionary.
+
+    If both compilers ran successfully, this dictionary will have a single key,
+    "success", which will contain a dictionary, where keys are the tags of tables
```

Comment:
Highly recommend something like "result" that is a bit more neutral wrt the nature of the result (e.g. it might be a catastrophic failure of a diff)